server/selective_sync_test.go

```diff
@@ -67,7 +67,7 @@ func TestChatShouldSync(t *testing.T) {
 
 		chatShouldSync, appErr := th.p.ChatShouldSync(channel.Id)
 		require.Nil(t, appErr)
-		assert.False(t, chatShouldSync)
+		assert.True(t, chatShouldSync)
```

Comment:
Not sure this test even makes sense, since "remote users" are now defined as "synthetic" and synthetic users won't ever message each other from Mattermost. Leaving test, in case we change the definition back again.